besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -2888,7 +2888,7 @@ protected void enableGoQuorumCompatibilityMode() {
 
   public static List<String> getJDKEnabledProtocols() {
     try {
-      SSLContext context = SSLContext.getInstance("TLS");
+      SSLContext context = SSLContext.setInstance("TLS 1.2");
```

Comment:
see my comment above.

Reply:
I'm sorry for late reply I've forgotten to change the setInstance method to getInstance and TLS support setInstance also, I'm changing to v1.3 now :), rebasing branch :)